src/Nethermind/Nethermind.Api/Extensions/IPluginLoader.cs

```diff
@@ -3,16 +3,14 @@
 
 using System;
 using System.Collections.Generic;
-using Nethermind.Logging;
 
-namespace Nethermind.Api.Extensions
+namespace Nethermind.Api.Extensions;
+
+public interface IPluginLoader
 {
-    public interface IPluginLoader
-    {
-        IEnumerable<Type> PluginTypes { get; }
+    IEnumerable<Type> PluginTypes { get; }
 
-        void Load(ILogManager logManager);
+    void Load();
```

Comment:
The `ILogManager` parameter has been removed from the `Load()` method. Implementations can require it in their constructors if needed. This was helpful in the context of `Program.cs` refactoring.